pysat/instruments/templates/template_instrument.py

```diff
@@ -352,8 +343,37 @@ def load(fnames, tag=None, sat_id=None, custom_keyword=None):
                                            max_label='ValidMax',
                                            fill_label='FillVal',
```

Comment:
These are wrong now.

Reply:
I updated the code and the comments in the area.